SS14.Admin/wwwroot/js/logs-grid.js

```diff
@@ -2,23 +2,18 @@
 const overlay = document.getElementById('side-panel-overlay');
 const entries = document.querySelectorAll('.data-log');
 
-let selecting = false;
-
-document.addEventListener('selectstart', () => selecting = true);

```

Comment:
The selecting stuff was for allowing you to select text inside the table.
I guess that is what caused opening the details panel to be finicky.